common/api/core-geometry.api.md

```diff
@@ -4326,6 +4326,7 @@ export abstract class Polyface extends GeometryQuery {
     data: PolyfaceData;
     get expectedClosure(): number;
     set expectedClosure(value: number);
+    abstract get facetCount(): number;
```

Comment:
This is a breaking change for any hypothetical subclasses of `Polyface`. Can the same functionality be achieved without breaking change?

Reply:
Thanks for the tip, @pmconne. I gave `Polyface.facetCount` a default implementation to avoid impacting existing subclasses. And noted this in NextVersion.md.